tests/test_healthsystem.py

```diff
@@ -193,6 +193,48 @@ def test_run_no_interventions_allowed(tmpdir, seed):
     assert not any(sim.population.props['mi_status'] == 'P')  # No cures
 
 
+@pytest.mark.slow
+def test_policy_has_no_effect_on_mode1(tmpdir, seed):
+    """Events ran in mode 1 should be identical regardless of policy assumed.
+    In policy "No Services", have set all HSIs to priority below lowest_priority_considered,
+    in mode 1 they should all be scheduled and delivered regardless"""
+
+    output = []
+    for i, policy in enumerate(["Naive", "Test Mode 1"]):
```

Comment:
This test seems great now.

Ok, we don't have to have them all, but how about also the "No Policy Policy", and the "Clinical Vulnerable" one. Just so we have all bases covered.